src/foam/u2/view/UnstyledTableView.js

```diff
@@ -122,7 +123,7 @@ foam.CLASS({
     {
       class: 'Boolean',
       name: 'editColumnsEnabled',
-      value: true,
+      value: false,
```

Comment:
```suggestion

```

Reply:
This is across the board disabling the column selection ... I think its safer to leave it on and fix permissioning the columns https://nanopay.atlassian.net/browse/NP-4822